brianlong/www.validators.app
# 183372541 stake explorer

#### What's this PR do?
- create stake explorer

#### How should this be manually tested?
- run `rails r script/gather_explorer_stake_accounts.rb` if you don't have explorer stake accounts
- check vote accounts page for stake explorer
- test filters
- test pagination and links
- check explorer index page
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183372541)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
